tests/idl/rapidjson_generator/.gitignore

```diff
@@ -0,0 +1,14 @@
+RapidJsonTest
+RapidJsonTestC.cpp
+RapidJsonTestC.h
+RapidJsonTestC.inl
+RapidJsonTestS.cpp
+RapidJsonTestS.h
+RapidJsonTestTypeSupport.idl
+RapidJsonTestTypeSupportC.cpp
+RapidJsonTestTypeSupportC.h
+RapidJsonTestTypeSupportC.inl
+RapidJsonTestTypeSupportImpl.cpp
+RapidJsonTestTypeSupportImpl.h
+RapidJsonTestTypeSupportS.cpp
+RapidJsonTestTypeSupportS.h
```

Comment:
You can simplify these with wildcards, [See here for an example](https://github.com/objectcomputing/OpenDDS/pull/1067/files#diff-6b7d8a364cd6f0d9c12a10c7d6b2cc70R1)

Reply:
If you use wildcards then you're writing them by hand.  Please use the script tools/scripts/update_gitignore.pl